lib/ansible/module_utils/basic.py

```diff
@@ -1255,7 +1280,7 @@ def _log_invocation(self):
             msg = 'Invoked'
 
         # 6655 - allow for accented characters
-        if isinstance(msg, unicode):
+        if not isinstance(msg, str):
             # We should never get here as msg should be type str, not unicode
             msg = msg.encode('utf-8')
```

Comment:
Yeah, sounds like a plan for now.  If we can, it's better to fix it correctly than to do something incorrect and forget that it's still a problem.

If we get to the place where it's the only failing test left we can revisit whether a stopgap is appropriate so that other pieces of code don't regress.